README.md

```diff
@@ -226,7 +227,13 @@ In certain cases (so far, only `Gradle` has been confirmed) the implementation e
 but doesn't provide interfaces for it to actually make its work. Luckily, it was possible to make few assumptions about
 IDEA's internal implementation and make it work. Such integration is very fragile and it's not immediately clear if it
 will affect any existing integrations and when it will break. For that reason there is a special option to
-`Enable Experimental Integrations` that can be enabled when desired and should prevent other integrations from breaking. 
+`Enable Experimental Integrations` that can be enabled when desired and should prevent other integrations from breaking.
+
+#### direnv
+You need `direnv` in your `PATH` environment variable. `direnv allow` is run automatically if needed.
+
+Note: On macOS, you may need to add `/usr/local/bin` to the system-wide `PATH` using `launchctl` because `PATH` is not set correctly when IntelliJ is launched from Dock or Spotlight.
```

Comment:
@cmmoran Thanks for the tips! Do you think this note is not necessary?

Reply:
Correct. As long as you have a `.zprofile` (zsh) or `.profile` or (bash) with `direnv` in the PATH, then the JetBrains toolbox of apps will pick up the PATH and `direnv` will operate properly.

Reply:
I see. Thanks!